utils/land-patterns.stanza

```diff
@@ -1477,12 +1481,29 @@ public defn make-n-pin-soic-landpattern (num-pins:Int,                 ; number
     ;----------------------------------------------------
     ; place the pads
     val grid-sz = Dims(Gmin(ipc) + x(pad-sz), pitch)
-    val locs    = to-tuple(grid-locs(num-pins / 2, 2, x(grid-sz), y(grid-sz), false))
+    var pin-position:True|False = false
+    if pin-numbering is RowMajorNumbering :
+      pin-position = true
```

Comment:
`var pin-position = (pin-numbering is RowMajorNumbering)`

Reply:
I left the type declaration to make it easier to read for new people